assignments/callbacks.js

```diff
@@ -25,28 +25,53 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
 }
+function getLength(arr, cb) {
+    return cb(arr.length)
+}
+getLength(items, function(length){
+    console.log(length)
+});
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
 }
+function last(arr, cb) {
+    return cb(arr[arr.length-1]);
+}
+last(items, function(lastIndex){
+    console.log(lastIndex);
+})
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
 }
+function sumNums(x, y, cb) {
+    return cb(x+y);
+}
+sumNums(4,5,function(addition) {
+    console.log(addition);
+})
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
 }
+function multiplyNums(x, y, cb) {
+    return cb(x*y);
+}
+multiplyNums(3, 7, function(multiplication){
+    console.log(multiplication);
+})
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  for (let i =0; i < list.length; i++){
+      if (list.includes(item)) {
+          return true;
+      } else {
+          return false;
+      }
+  }
```

Comment:
Nice job with this! It can be refactored using a ternary operator:
```
function contains(item, list, cb) {
  return list.includes(item) ? true : false
  }
```
It's the same as your function above 😄 
It reads "Does the `list` include the `item`? If so, return `true`. If not return `false`"